cli/kubectl-hyperkube

```diff
@@ -194,6 +194,24 @@ def get_pem(ctx, cluster):
         print(f'Request error: {err}')
 
 
+@cli.command()
+@click.option('--cluster', '-g')
+@click.pass_context
+def remove_pem(ctx, cluster):
+    """Removethe pem file for a specific cluster"""
+
+    try:
+        r = requests.delete(
+            (f'{ctx.obj["url"]}/{ctx.obj["stage"]}'
+             f'/clusters/remove-pem?cluster_name={cluster}'),
+            headers=ctx.obj['headers']
+        )
+        if r.status_code == 404:
+            sys.exit(1)
```

Comment:
Should this fail with a message?

Reply:
Also, Why would it return a 404? It seems to only return a 200 or 500. https://github.com/zillow/hyper-kube-config/blob/556af4146db7d913d38368bd53c16544af74c331/pem.py#L93

Reply:
500 makes sense